src/System.Security.Cryptography.Cng/src/Configurations.props

```diff
@@ -2,9 +2,9 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
+      netstandard;
```

Comment:
I figured a PSNE assembly may as well be netstandard instead of netcoreapp-unix, so the assembly ends up under lib as platform-agnostic now.